osu.Game.Rulesets.Osu/Skinning/Argon/ArgonMainCirclePiece.cs

```diff
@@ -62,7 +62,10 @@ public ArgonMainCirclePiece(bool withOuterFill)
             {
                 outerFill = new Circle // renders white outer border and dark fill
```

Comment:
Tangential but isn't this wrong? This doesn't render any border, that'd be the `RingPiece` further down.

```suggestion
                outerFill = new Circle // renders dark fill
```

Had me mighty confused for a good second there.